tools/BUILD

```diff
@@ -11,10 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+load("//python:defs.bzl", "py_binary")
+
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
+# Implementation detail of py_wheel rule.
+py_binary(
+    name = "wheelmaker",
+    srcs = ["wheelmaker.py"],
+    visibility = ["//visibility:public"],
```

Comment:
The package default is `//visibility:public` so maybe this is redundant.

Reply:
Removed.